src/components/MonacoEditor/SourceCodeHeading.tsx

```diff
@@ -45,7 +46,7 @@ const SourceCodeHeading: React.FC<IProps> = ({
     // eslint-disable-next-line
   }, [sourceCodeTitle]);
 
-  function handleShowOptions(event: React.MouseEvent<HTMLButtonElement>) {
+  function handleShowOptions(event: any) {
```

Comment:
Same as above

Reply:
yeah, it ought to be, but typescript was throwing an error that it was a wrong type